composer.json

```diff
@@ -29,6 +29,9 @@
   "require": {
     "consolidation/output-formatters": "^3.2.0",
     "drupal/dynamic_entity_reference": "^2.0@alpha",
-    "drupal/encrypt": "^3.0@rc"
+    "drupal/encrypt": "^3.0@rc",
+    "drupal/key": "^1.7",
+    "lusitanian/oauth": "^0.8.11",
+    "firebase/php-jwt": "^5.0"
```

Comment:
Doesn't look like JWT is being implemented in this phase. Probably harmless but don't need to require yet. Didn't see Key used either.